lib/helpers.js

```diff
@@ -12,19 +12,15 @@ const crypto = require('crypto');
 const {documentLoader} = require('bedrock-jsonld-document-loader');
 const getRandomBytes = promisify(crypto.randomBytes);
 const jsigs = require('jsonld-signatures');
-const {extendContextLoader, SECURITY_CONTEXT_V2_URL} = jsigs;
-const {Ed25519Signature2018} = jsigs.suites;
+const {extendContextLoader} = jsigs;
+const {CONTEXT_URL: ZCAP_CONTEXT_URL} = require('zcap-context');
```

Comment:
This can be had by doing on L20, and then you can likely remove the `zcap-context` dep.
```
const {CapabilityDelegation, constants: {ZCAP_CONTEXT_URL}} =
  require('@digitalbazaar/zcapld');
```

Reply:
Updated here 030dd5e